{
  "about": "https://jvndb.jvn.jp/ja/contents/2018/JVNDB-2018-006702.html",
  "title": "PBC におけるバッファエラーの脆弱性",
  "link": "https://jvndb.jvn.jp/ja/contents/2018/JVNDB-2018-006702.html",
  "description": "PBC には、バッファエラーの脆弱性が存在します。",
  "identifier": "JVNDB-2018-006702",
  "references": [
    {
      "text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-12916",
      "source": "CVE",
      "id": "CVE-2018-12916"
    },
    {
      "text": "https://nvd.nist.gov/vuln/detail/CVE-2018-12916",
      "source": "NVD",
      "id": "CVE-2018-12916"
    },
    {
      "text": "https://github.com/cloudwu/pbc/issues/120",
      "source": "関連文書",
      "id": "Segmentation fault in _pbcP_message_default () #120"
    },
    {
      "text": "https://jvndb.jvn.jp/ja/cwe/CWE-119.html",
      "id": "CWE-119",
      "title": "バッファエラー(CWE-119)"
    }
  ],
  "cpe": [
    {
      "text": "cpe:/a:pbc_project:pbc",
      "version": "2.2",
      "vendor": "PBC project",
      "product": "PBC"
    }
  ],
  "cvss": [
    {
      "version": "2.0",
      "score": "7.5",
      "type": "Base",
      "severity": "High",
      "vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
    },
    {
      "version": "3.0",
      "score": "9.8",
      "type": "Base",
      "severity": "Critical",
      "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
    }
  ],
  "date": "2018-08-29T17:32+09:00",
  "issued": "2018-08-29T17:32+09:00",
  "modified": "2018-08-29T17:32+09:00"
}
